### DISCUSSION TITLE
is this project still active?

### DISCUSSION THREAD

**Discussion by codefish1** [Siouan/frontend-gradle-plugin#191] [General]
We've come across this project which looks wicked thanks for it.  However it looks inactive and the author hasn't contributed since Sept last year to Github at all.

What is the current state on it ?

**Comment by v1nc3n4** [comment]
Hi @codefish1,

Thank you for your message.
I understand perfectly the problem/questions you are facing with the apparent inactivity of the project and the impact on production apps relying on it. The project is still alive (see [this discussion](https://github.com/siouan/frontend-gradle-plugin/issues/183) end of 2022), but I struggled past months to find enough time to complete some major enhancements, and [an issue](https://github.com/nodejs/corepack/issues/178) with Corepack blocked an important and awaited release I'm still waiting an answer for from Node.js team. I must admit I didn't have time also to get in touch with this problem and, maybe, it is now resolved though the issue is still opened.

I will provide concrete news May 2023 about what will come next. Thanks again for your consideration.
BR

**Comment by codefish1** [comment]
thanks for the quick reply. 

I assume you are willing to accept PRs to help fix some of the issues.  We are looking to migrate from using Maven with frontend-maven-plugin and this looked like a good choice.

**Reply by v1nc3n4** [reply]
Yes, PR are welcome. Feel free to discuss about your difficulties, if any, before implementing a fix. There may be some misunderstandings or workarounds that would avoid you lost time, or simply require a documentation update.
Anyways, thank you for your intention!
